manifests/1.1.1/opensearch-dashboards-1.1.1.yml

```diff
@@ -0,0 +1,49 @@
+---
+schema-version: "1.0"
+ci:
+  image:
+     name: "opensearchstaging/ci-runner:centos7-x64-arm64-jdkmulti-node10.24.1-cypress6.9.1.10211.18"
+build:
+  name: OpenSearch Dashboards
+  version: 1.1.1
+components:
```

Comment:
Is everybody rebuilding? If not start with dist like this: https://github.com/opensearch-project/opensearch-build/blob/main/manifests/1.1.1/opensearch-1.1.1.yml#L16

Reply:
1.1.0 dashboards does not have dist.
It is built with another script that requires plugin owner to provide zips.

Reply:
I think we should use the latest OpenSearch dashboards build that was automated for this, just to exercise the intent and the mechanism. We were building it not by hand!

Reply:
I dont plan to build manually again.
That is why I point to the source not the dist.
Because there is not a known good build as of now.

If we have 1.1.2 next time we can use the 1.1.1 release dist build version as reference.

Reply:
This doesn't sound right. We have a set of binaries that we _have_ shipped and those were produced manually. I think we should use those for 1.1.1 for the components that don't change.

Reply:
They are not in the same S3 bucket as the current CICD and they are not even having the same folder structure.
If you know how the syntax work let me know.

Thanks.

Reply:
Yes, that's correct.

Both the build and dist manifest for OpenSearch Dashboards are very simple. I would take any build of OpenSearch dashboards, put in a new folder, and replace the .tar.gz and .zip files with GA ones. The only change in the manifest(s) would be the `commit_id` values, which can just be removed as they aren't used for building.

Building everything from source is IMHO net worse, because we break the customer guarantee that nothing changed.

Reply:
Got it now. Will try that early next week.

Thanks.

Reply:
Fixed.